includes/payment-methods/class-wc-stripe-upe-payment-method-klarna.php

```diff
@@ -112,7 +125,7 @@ public function get_supported_currencies() {
 
 		// Stripe account in EEA + UK and Switzerland can present the following as store currencies.
 		// EEA currencies can only transact with countries where that currency is the standard currency.
-		return [ 'CHF', 'CZK', 'DKK', 'EUR', 'GBP', 'NOK', 'PLN', 'SEK' ];
+		return [ WC_Stripe_Currency_Code::SWISS_FRANC, WC_Stripe_Currency_Code::CZECH_KORUNA, WC_Stripe_Currency_Code::DANISH_KRONE, WC_Stripe_Currency_Code::EURO, WC_Stripe_Currency_Code::POUND_STERLING, WC_Stripe_Currency_Code::NORWEGIAN_KRONE, WC_Stripe_Currency_Code::POLISH_ZLOTY, WC_Stripe_Currency_Code::SWEDISH_KRONA ];
```

Comment:
What do you think about breaking up this now-superlong line into multiple lines?

Reply:
Thanks for your review, Anne! I have updated both currency lists in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/3459/commits/3fcebaa1f4282aed05d73be75c39c7da6b374447